### ISSUE TITLE
How to unhide item from feed, if it was "deleted" manually? 

### ISSUE THREAD

**Issue by Larissa-Kravtsova** [Xyrio/RSSOwlnix#153]
How to unhide item from feed, if it was "deleted" manually?

OS: win10
RSSOwlnix version: RSSOwlnix-2-9-0-win32.win32.x86_64

**Comment by Xyrio** [comment]
there is no function for that. if you just deleted it you can undo.
figure out yourself how to edit the database directly ( http://www.softsea.com/review/db4o-Object-Manager.html ).
maybe cleanup deleted stuff then redownload rss. or write your own rss and add it with `file:///` prefix like `file:///c:/data.rss`

**Comment by Larissa-Kravtsova** [comment]
> figure out yourself how to edit the database directly ( http://www.softsea.com/review/db4o-Object-Manager.html ).

Yeah, I've already tried. After each edit something breaks. I just change StateOrdinal from 4 (HIDDEN) to 2, and then all news are gone, just empty feeds.

> you can add stuff to links using `#1 #2 ...` like `http://google.com/the.rss#1` `http://google.com/the.rss#2` to make them unique but still working the same.

Thanks for the tip! I'll use this probably.
